ui/client/containers/Processes.js

```diff
@@ -138,6 +140,8 @@ class Processes extends BaseProcesses {
           isSubprocess={false}
           visualizationPath={Processes.path}
           message="Create new process"
+          processes={this.state.processes}
```

Comment:
usedProcessNames, takenProcessNames?

Reply:
It contains info about processes, not only names. I think that processes is the shortest acceptable variable name.